apiserver/admin.go

```diff
@@ -127,6 +127,10 @@ func (a *admin) login(ctx context.Context, req params.LoginRequest, loginVersion
 		a.srv.facades,
 		a.root.resources,
 		a.root,
+		httpRequestRecorderWrapper{
```

Comment:
Can we also change this method signature so that it doesn't need to be pass both `a.root` and 3 members of `a.root`? That's shit.